src/components/dialogs/confirm-name-request.vue

```diff
@@ -34,11 +34,17 @@
           </v-tab-item>
 
           <v-tab-item>
-            <v-card-title class="d-flex justify-space-between">
-              <div>Confirm Name Request</div>
-              <v-btn icon large class="dialog-close float-right" @click="hideModal()">
-                <v-icon>mdi-close</v-icon>
-              </v-btn>
+            <v-card-title>
+              <v-row no-gutters>
+                <v-col cols="8">
+                  <span>Confirm Name Request</span>
+                </v-col>
+                <v-col>
+                  <v-btn icon large class="dialog-close" @click="hideModal()">
+                    <v-icon>mdi-close</v-icon>
+                  </v-btn>
+                </v-col>
```

Comment:
I'm not sure that the new code is equivalent to the old code.
The old code floated the button all the way to the right.
The new code puts the button in the last 4 columns... how does this look on a wide screen?